File: python/TestFor16.py
def isDivBy16(n):
	last_four_digits = int(n[-4:])
	result = last_four_digits / 16
	if result.is_integer():
		return True
	else:
		return False


for i in range(10000):
	print(f'{i} is divisible by 16? {isDivBy16(str(i))}')

File: lua/TestFor16.lua
function divide_by_16(value)
    result = value/16
    if result == math.floor(result) then
        return true
    end
    return false
end

for val=1,100 do
    print(string.format("%d divisible by 16?: %s",val,divide_by_16(val)))
end
